Migrate docs to the new kyma cli (#1542)
kyma-project/serverless: docs/user/tutorials/01-30-manage-functions-with-kyma-cli.md

This tutorial shows how to use the available CLI commands to manage Functions in Kyma. You will see how to:

1. Generate local files that contain source code and dependencies for a sample "Hello World" Python Function (`kyma alpha function init`).
2. Create a Function custom resource (CR) from these files and apply it on your cluster (`kyma alpha function create`).

> 3. List all Function custom resources from the cluster (`kyma alpha function get`).

Reviewer: ```suggestion
3. List all Function CRs from the cluster (`kyma alpha function get`).
```

> 4. Delete previously created Function custom resource from the cluster (`kyma alpha function delete`).

Reviewer: ```suggestion
4. Delete previously created Function CR from the cluster (`kyma alpha function delete`).
```

  The `init` command creates these files in your workspace folder:

- `handler.py` - the Function's code and the simple "Hello World" logic
- `requirements.txt` - an empty file for your Function's custom dependencies

>   After the command is succeeded you will see the following output message with suggestions on how you can run the code:

Reviewer: ```suggestion
  After your files are successfully created, you will see the following output message with suggestions on how you can run the code:
```

  ```text
  Next steps:
  * update output files in your favorite IDE
  * create Function, for example:
    kyma alpha function create python312 --runtime python312 --source handler.py --dependencies requirements.txt
  ```

  > [!NOTE]
  > Now is a good time to customize your Python runtimes source code. You can do it by editing generated files in your favorite editor.

2. Run the `create` Kyma CLI command to create a Function CR in the YAML format in your cluster based on the suggestion from the previous command:

  ```bash
  kyma alpha function create python312 --runtime python312 --source handler.py --dependencies requirements.txt
  ```

> 3. Once applied, view the Function's state in the cluster:

Reviewer: ```suggestion
3. Once applied, check the Function's state in the cluster:
```
